tools/crm_mon_print.c

```diff
@@ -1249,18 +1246,10 @@ print_node_attributes(mon_state_t *state, pe_working_set_t *data_set, unsigned i
 static void
 print_cluster_summary_header(mon_state_t *state)
 {
-    switch (state->output_format) {
-        case mon_output_html:
-        case mon_output_cgi:
-            fprintf(state->stream, " <h2>Cluster Summary</h2>\n <p>\n");
-            break;
-
-        case mon_output_xml:
-            fprintf(state->stream, "    <summary>\n");
-            break;
-
-        default:
-            break;
+    if (state->output_format == mon_output_xml) {
+        state->out->message(state->out, "begin-parent", "summary");
+    } else {
+        state->out->message(state->out, "header", 2, "Cluster Summary");
```

Comment:
I originally thought the header message would be more broadly useful, but it turns out I'm only using it in one or two places.  So that can just go away entirely.  I'll look at further changing the cluster summary messages.